static/js/timerender.js

```diff
@@ -307,29 +307,38 @@ export const absolute_time = (function () {
 })();
 
 export function get_full_datetime(time) {
-    // Convert to number of hours ahead/behind UTC.
-    // The sign of getTimezoneOffset() is reversed wrt
-    // the conventional meaning of UTC+n / UTC-n
-    const tz_offset = -time.getTimezoneOffset() / 60;
+    const date_string_options = {weekday: "long", month: "long", day: "numeric"};
+    const time_string_options = {timeStyle: "full"};
```

Comment:
After some discussion with @alya, I think we should try the 1-line:

"{date} at {time}" format, using the shorter timezone indications of `PST (UTC+N)`.  Given the fact that we are displaying the time in the user's own timezone always, it is probably unnecessary to show full timezone string here.